### ISSUE TITLE
[Feature Request] Tabs like internet browsers for chats 

### ISSUE THREAD

**Issue by matinkg** [telegramdesktop/tdesktop#24676]
### Is your feature request related to a problem?

No

### Describe the solution you'd like

It could be better to open chats in tabs like chrome and firefox. Currently, we can open chats in new windows. We can merge them and switch between them in only one window.

### Describe alternatives you've considered

-

**Comment by ilya-fedin** [comment]
How it differs from switching between chats in chat list? :thinking:

**Comment by FinecoFinit** [comment]
> How it differs from switching between chats in chat list? :thinking: 

Another way to multitasking

**Comment by FinecoFinit** [comment]
> Why chat list is not enough? :thinking: 

Maybe @axone13 talking about thin window, when chat list is not presented, well, this issue can be solved with chat avatars always showed

**Comment by axone13** [comment]
> How it differs from switching between chats in chat list? 🤔

Hi @ilya-fedin
When we use the chat list, we are switching between all the chats and if we want to switch between two chats that are far from each other in the chat list, we should either search them each time or pin them to the top. 
In the last telegram desktop update, we can hold control to open chats in new windows. This way, we can easily switch between opened chats by switching between their windows. When the number of open chats increases, it becomes very hard to switch between the windows. So it's much better to have only one window for opened chats; and if we open a new chat, it adds to that window as a new tab.
We can also use another keybinding for opening chats in the new tab. For example ```ctrl + opening chat``` open a chat in a new window, and ```shift + opening chat``` open a chat in a new tab.

**Comment by ilya-fedin** [comment]
Isn't pins what you want?

**Comment by axone13** [comment]
> Isn't pins what you want?

We usually use pins for important chats and it's kinda permanent. But most of the time we may want to chat with many people at the same time and we don't want to pin them permanently on top. If this option was added to the telegram, we could open their chats as tabs and switch between them. And when are chatting is finished, close their tabs.

![image](https://user-images.githubusercontent.com/47145617/175260052-3d989142-b21a-4f6e-b83f-f873c6ef824f.png)

**Comment by Posya** [comment]
Pins and list of chats does not helps if you use chat list folders and view, for example, channels in another folder.

**Comment by esouldy** [comment]
> How it differs from switching between chats in chat list? 🤔

when you have a some ACTIVE chats and many read-only chats, any message in r/o chat move active and needed chats to bottom

**Comment by ilya-fedin** [comment]
@esouldy you can use chat folders for that

**Comment by esouldy** [comment]
> @esouldy you can use chat folders for that

hmmm...sounds like "you can use bookmarks in your browser instead tabs" =)
Folders are STATIC groups of chats (work, family, news etc). And tabs are chats, which you need right now, and it isn't static list.

**Comment by esouldy** [comment]
no no, pinned chats, folders - are ok. But, if a need to switch between some active chats - tabs are best solution. Different windows looks like  "hello from '00 from internet explore..." =)

**Comment by axone13** [comment]
> no no, pinned chats, folders - are ok. But, if a need to switch between some active chats - tabs are best solution. Different windows looks like "hello from '00 from internet explore..." =)

I agree with you, I don't know why they resist. Currently opening chats in new windows is implemented (by holding ctrl + clicking on a chat).  We just need tabs to group them together. 
Currently, telegram is like windows 10 file explorer. We can only open chats in new windows. But even Microsoft added tabs to windows 11 file explorer. 
I think tabs is the only feature that telegram desktop doesn't have yet.

**Comment by ilya-fedin** [comment]
> I don't know why they resist

Well, you're free to implement this if it's something you really want
